Rebuilt the newsletter sign-up page (#3318)
PostHog/posthog.com: contents/newsletter.md

showTitle: true
---

![PostHog Newsletter](images/docs/newsletter/newshogs.png)

<Section
    divider={false}

>     title="HogMail is a fortnightly newsletter that’s about helping you make better products."

Reviewer: ```suggestion
    title="HogMail is a fortnightly newsletter about helping you make better products."
```

>     title="HogMail is a fortnightly newsletter that’s about helping you make better products."

Reviewer: ```suggestion
    title="HogMail is a fortnightly newsletter about helping you make better products."
```

>     title="HogMail is a fortnightly newsletter that’s about helping you make better products."

Reviewer 2: As an American, I'm not quite sure what "fortnightly" means! =]

Reviewer: Good to know! Had no idea you guys didn't use the term. 

It means every two weeks, or bi-weekly. But bi-weekly is super confusing because it can mean twice a week or every two weeks!

Maybe for now we just drop the timeframe thing entirely?

    size="full"
    cols={2}
>
    <div>
        <ul className="text-[20px] font-semibold">

>             <li className="text-[20px]">Select cuts from the PostHog blog</li>
>             <li className="text-[20px]">The latest tutorials and news</li>

Reviewer 3: ```suggestion
            <li>Select cuts from the PostHog blog</li>
            <li>The latest tutorials and news</li>
```

not affecting the `font-size` with `text-[20px]` (even you add it), and set to inherited font-size `17px` (as having high priority), which feels ok

        </ul>
    </div>
    <div>
        <ul className="text-[20px] font-semibold">
            <li>A selections of new articles we love</li>

>             <li>And an <span className="text-red">awesome hedgehog cartoon!</span></li>

Reviewer: ```suggestion
            <li>And an <strong>awesome hedgehog cartoon!</strong></li>
```

We should avoid using the red/orange text, as that's what we use for links.

>             <li className="text-[20px]">A selections of new articles we love</li>
>             <li className="text-[20px]">And an <span className="text-red">awesome hedgehog cartoon!</span></li>

Reviewer 3: ```suggestion
            <li>A selections of new articles we love</li>
            <li>And an <span className="text-red">awesome hedgehog cartoon!</span></li>
```
not affecting the `font-size` with `text-[20px]`,same as above
